packs/parrotparty.yaml

```diff
@@ -1,63 +1,75 @@
+    src: http://cultofthepartyparrot.com/parrots/coffeeparrot.gif
+  - name: confusedparrot
+    src: http://cultofthepartyparrot.com/parrots/confusedparrot.gif
   - name: congaparrot
     src: http://cultofthepartyparrot.com/parrots/congaparrot.gif
-  - name: reversecongaparrot
-    src: http://cultofthepartyparrot.com/parrots/reversecongaparrot.gif
-  - name: partyparrot
-    src: http://cultofthepartyparrot.com/parrots/partyparrot.gif
-  - name: sadparrot
-    src: http://cultofthepartyparrot.com/parrots/sadparrot.gif
-  - name: parrotcop
-    src: http://cultofthepartyparrot.com/parrots/parrotcop.gif
-  - name: fastparrot
-    src: http://cultofthepartyparrot.com/parrots/fastparrot.gif
-  - name: slowparrot
-    src: http://cultofthepartyparrot.com/parrots/slowparrot.gif
-  - name: parrotdad
-    src: http://cultofthepartyparrot.com/parrots/parrotdad.gif
+  - name: congapartyparrot
+    src: http://cultofthepartyparrot.com/parrots/congapartyparrot.gif
+  - name: darkbeerparrot
+    src: http://cultofthepartyparrot.com/parrots/darkbeerparrot.gif
   - name: dealwithitparrot
     src: http://cultofthepartyparrot.com/parrots/dealwithitparrot.gif
+  - name: dreidelparrot
+    src: http://cultofthepartyparrot.com/parrots/dreidelparrot.gif
+  - name: explodyparrot
+    src: http://cultofthepartyparrot.com/parrots/explodyparrot.gif
+  - name: fastparrot
+    src: http://cultofthepartyparrot.com/parrots/fastparrot.gif
+  - name: fieriparrot
+    src: http://cultofthepartyparrot.com/parrots/fieriparrot.gif
   - name: fiestaparrot
     src: http://cultofthepartyparrot.com/parrots/fiestaparrot.gif
-  - name: pizzaparrot
-    src: http://cultofthepartyparrot.com/parrots/pizzaparrot.gif
+  - name: gentlemanparrot
+    src: http://cultofthepartyparrot.com/parrots/gentlemanparrot.gif
+  - name: gothparrot
+    src: http://cultofthepartyparrot.com/parrots/gothparrot.gif
   - name: hamburgerparrot
     src: http://cultofthepartyparrot.com/parrots/hamburgerparrot.gif
-  - name: chillparrot
-    src: http://cultofthepartyparrot.com/parrots/chillparrot.gif
-  - name: explodyparrot
-    src: http://cultofthepartyparrot.com/parrots/explodyparrot.gif
-  - name: shufflepartyparrot
-    src: http://cultofthepartyparrot.com/parrots/shufflepartyparrot.gif
-  - name: parrot
+  - name: harrypotterparrot
+    src: http://cultofthepartyparrot.com/parrots/harrypotterparrot.gif
+  - name: ice-cream-parrot
     src: http://cultofthepartyparrot.com/parrots/ice-cream-parrot.gif
-  - name: sassyparrot
-    src: http://cultofthepartyparrot.com/parrots/sassyparrot.gif
-  - name: confusedparrot
-    src: http://cultofthepartyparrot.com/parrots/confusedparrot.gif
-  - name: aussiecongaparrot
-    src: http://cultofthepartyparrot.com/parrots/aussiecongaparrot.gif
-  - name: aussiereversecongaparrot
-    src: http://cultofthepartyparrot.com/parrots/aussiereversecongaparrot.gif
+  - name: magaritaparrot
+    src: http://cultofthepartyparrot.com/parrots/magaritaparrot.gif
```

Comment:
@woganmay FYI the PR to fix the spelling has been merged. This will affect both Margarita Parrot as well as Blues Clues Parrot.
"magarita" -> "margarita"
"blueclues" -> "bluesclues"